defaults/main.yml

```diff
@@ -1,4 +1,12 @@
 ---
+
+jitsi_meet_packages:
```

Comment:
these are rather constants than defaults and imo should hence be kept and loaded from a `vars/main.yml`.

Reply:
you might want to override this to install e.g. `jitsi-meet-turnserver`

Reply:
bat that'd be an additional option that either belongs to a dedicated role or should be controlled w/ a boolean variable.